src/main/java/com/coreos/jetcd/EtcdLease.java

```diff
@@ -0,0 +1,40 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.api.LeaseGrantResponse;
+import com.coreos.jetcd.api.LeaseKeepAliveResponse;
+import com.coreos.jetcd.api.LeaseRevokeResponse;
+import com.google.common.util.concurrent.ListenableFuture;
+
+/**
+ * Interface of Lease talking to etcd
+ */
+public interface EtcdLease {
+
+    /**
+     * New a lease with ttl value
+     * @param ttl ttl value
+     * @return
+     */
+    ListenableFuture<LeaseGrantResponse> grant(long ttl);
+
+    /**
+     * revoke one lease and the key bind to this lease will be removed
+     * @param leaseId the id of lease
+     * @return
+     */
+    ListenableFuture<LeaseRevokeResponse> revoke(long leaseId);
+
+    /**
+     * keep alive one lease in background
+     * @param leaseId
+     */
+    void keepAlive(long leaseId);
```

Comment:
Compared with the Go definition:
```
KeepAlive(...) (<-chan *LeaseKeepAliveResponse, error)
```
How does user get the response of each heartbeat?